skymp5-server/cpp/server_guest_lib/PapyrusKeyword.cpp

```diff
@@ -0,0 +1,39 @@
+#include "PapyrusKeyword.h"
+
+VarValue PapyrusKeyword::GetKeyword(VarValue self,
+                                    const std::vector<VarValue>& arguments)
+{
+  if (arguments.empty()) {
+    spdlog::error("Keyword.GetKeyword - at least one argument expected");
+    return VarValue();
+  }
+
+  std::string keywordName = arguments[0].GetType() == VarValue::kType_String
```

Comment:
std::string -> CIString, then remove `std::transfprm`
And case-insensetive == check will be done automatically, without `std::transform`

it'd be much faster